### PR TITLE
Adapt active-response definition

### LINKED ISSUE #223: Wazuh Release 3.11.4_7.6.1
Wazuh version: 3.11.4
Elastic version: 7.6.1

### Tasks

- [x] Adapt to new versions (3.11.4 - 7.6.1)

- [x] Check templates and update them if needed

- [ ] Tests

- [x] Update changelog

- [ ] Tag: v3.11.3_7.5.2

- [ ] Generate and publish the module on Puppet Forge

- [ ] Update documentation references

- [ ] Draft release

### LINKED ISSUE #233: Adapt the way active-response manifest and template are called
Hi all!

This issue will describe the changes needed to solve the issue commented in [Wazuh Slack Thread](https://wazuh.slack.com/archives/C0A933R8E/p1584874423281700).

`manager.pp` and `agent.pp` are not using `wazuh::activeresponse` when adding the corresponding `active-response` configuration in `ossec.conf`.

Kr,

Rshad

### PR DESCRIPTION
**Pull request by rshad**
Hi all!

This PR resolves #233 

Kr,

Rshad

### CODE REVIEW

File: manifests/activeresponse.pp [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
 #Define for a specific ossec active-response
 define wazuh::activeresponse(
-  $command_name,
-  $ar_location           = 'local',
-  $ar_level              = 7,
-  $ar_agent_id           = '',
-  $ar_rules_id           = [],
-  $ar_timeout            = 300,
-  $ar_repeated_offenders = '',
+  $active_response_name               = 'Rednering active-response template',
```
Comment (jm404) [inline manifests/activeresponse.pp]: Re**n**dering active-response template

File: manifests/agent.pp [hunk 2]
```diff
@@ -187,10 +187,17 @@
...
   # active-response
-  $ossec_active_response_disabled          =  $wazuh::params_agent::active_response_disabled,
-  $ossec_active_response_linux_ca_store    =  $wazuh::params_agent::active_response_linux_ca_store,
-  $ossec_active_response_windows_ca_store  =  $wazuh::params_agent::active_response_windows_ca_store,
-  $ossec_active_response_ca_verification   =  $wazuh::params_agent::active_response_ca_verification,
+  $ossec_active_response_disabled             =  $wazuh::params_agent::active_response_disabled,
+  $ossec_active_response_linux_ca_store       =  $wazuh::params_agent::active_response_linux_ca_store,
+  
+  $ossec_active_response_ca_verification      =  $wazuh::params_agent::active_response_ca_verification,
+  $ossec_active_response_command              =  $wazuh::params_manager::active_response_command,
```
Comment (jm404) [inline manifests/agent.pp]: Why using a parameter from `params_manager`?

Comment (rshad) [reply manifests/agent.pp]: It's wrong, it should be `params_agent`. Thanks for reporting. 
https://github.com/wazuh/wazuh-puppet/pull/234/commits/701a7e45f8999b127b440c425eabafd3cda50f8c

File: manifests/activeresponse.pp (hunk 1 above)
Comment (rshad) [reply manifests/activeresponse.pp]: Fixed in. https://github.com/wazuh/wazuh-puppet/pull/234/commits/701a7e45f8999b127b440c425eabafd3cda50f8c

**Pull request merged**